Sources/PointFree/PageLayout.swift

```diff
@@ -341,20 +341,3 @@ private let prismJsHead: [Html.ChildOf<Html.Tag.Head>] = [
     """
   )
 ]
-
-private func downgrade(node: HtmlUpgrade.Node) -> [Html.Node] {
```

Comment:
made this public and moved to `Styleguide` so that more modules could get access to it.